components/schema-migrator/dump_db.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+NC='\033[0m' # No Color
+
+ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+set -e
+
+if [[ -f ${ROOT_PATH}/seeds/dump.sql ]]; then
+    echo -e "${GREEN}Will reuse existing DB dump in schema-migrator/seeds/dump.sql ${NC}"
+else
+    echo -e "${YELLOW}Warning: Please ensure that your kubectl context points to an existing Compass development cluster${NC}"
+    sleep 2
+
+    CURRENT_KUBE_CONTEXT=$(kubectl config current-context)
+    if [[ $CURRENT_KUBE_CONTEXT != *dev* ]]; then
+        echo -e "${RED}Error: Current kubectl context does not point to an existing Compass development cluster${NC}"
+        exit 1
+    fi
+
+    echo -e "${GREEN}Will dump and use database data from connected kubernetes Compass development cluster${NC}"
+
+
+    REMOTE_DB_PWD=$(base64 -d <<< $(kubectl get secret -n compass-system compass-postgresql -o=jsonpath="{.data['postgresql-director-password']}"))
+    DIRECTOR_POD_NAME=$(kubectl get pods -n compass-system | grep "director" | head -1 | cut -c -33)
```

Comment:
cut command at the end ensure we've got the correct director pod? if for some reason we get the wrong pod containing director in the name e.g.compass-director-clients-scopes-synchronization-llrk7?

Reply:
Cut -33 ensures that we only get the name of the pod, otherwise there's some unnecessary junk till the end of the line:
![image](https://user-images.githubusercontent.com/15074116/119654781-cf275d00-bdf6-11eb-9ba6-9e76ca13d0ea.png)